core/modules/date/tests/date_api.test

```diff
@@ -142,7 +142,8 @@ class DateAPITestCase extends BackdropWebTestCase {
     // Create date object from datetime string.
     $input = '2009-03-07 10:30';
     $timezone = 'America/Chicago';
-    $date = new dateObject($input, $timezone);
+    //$date = new dateObject($input, $timezone);
+    $date = new BackdropDateTime($input, $timezone);
```

Comment:
@quicksketch It was complaining that there is now such class dateObject.
I looked in `date.inc` and used `BackdropDateTime` objects instead.